Makefile

```diff
@@ -40,84 +40,57 @@ cri_version   =  release-1.26
 # Configuration Options
 export GIT_PAGER = cat
 
+#------------------------------------------------------------------------------#
+
+# Convenience commands
 
 .PHONY: all
 all: install ## Build and install (debug) 🎉
 
-.PHONY: install
-install: build ## Build and install (debug) 🎉
-
 .PHONY: build
-build: musl auraed auraescript ## Build and install (debug) (+musl) 🎉
+build: install ## Build and install (debug) (+musl) 🎉
```

Comment:
is this just an alias for `install`?  i would expect install to move the binaries to /usr/local/bin or something .. do we need both?

Reply:
Yes, it is just an alias. `install` installs the binaries at, for example, `$(HOME)/.cargo/bin/auraed`.

I'll update build to not run the tests, and have install run tests. I think that's a good middle ground.